app/scrcpy.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=scrcpy
+GenericName=Android Remote Control
+Comment=Display and control your Android device
+Exec=scrcpy
+Icon=gparted
```

Comment:
![scrcpy](https://user-images.githubusercontent.com/543275/46915241-18f60880-cfa9-11e8-9c1d-bc247d525fb4.png)

_gparted_ icon does not really fit _scrcpy_. Unfortunately, I didn't find any existing icon that could fit. Maybe we should add an Android icon?

Reply:
You can use this icon ![guiscrcpy](https://raw.githubusercontent.com/srevinsaju/guiscrcpy/master/ui/guiscrcpy_logo.png), which I created for guiscrcpy. I licensed it under Creative Commons Attributions 4.0, so you can easily reuse it.

Reply:
The Android icon at https://github.com/Genymobile/scrcpy/blob/v1.18/app/src/icon.xpm may be a better choice. Here is an improved patch:
```Diff
diff --git a/app/meson.build b/app/meson.build
index 0663c64..321ed38 100644
--- a/app/meson.build
+++ b/app/meson.build
@@ -148,6 +148,14 @@ executable('scrcpy', src,
 
 install_man('scrcpy.1')
 
+# Desktop entry file for application launchers
+if host_machine.system() == 'linux'
+    # -> /usr/share/applications/scrcpy.desktop
+    install_data('scrcpy.desktop', install_dir : join_paths(get_option('datadir'), 'applications'))
+    install_data('src/icon.xpm', rename: 'scrcpy.xpm',
+                 install_dir : join_paths(get_option('datadir'), 'icons', 'hicolor', '48x48', 'apps'))
+endif
+
 
 ### TESTS
 
diff --git a/app/scrcpy.desktop b/app/scrcpy.desktop
new file mode 100644
index 0000000..5933272
--- /dev/null
+++ b/app/scrcpy.desktop
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=scrcpy
+GenericName=Android Remote Control
+Comment=Display and control your Android device
+Exec=scrcpy
+Icon=scrcpy
+Terminal=false
+Type=Application
+Categories=Utility;RemoteAccess;
+StartupNotify=false
```